implementations/rust/ockam/ockam_app/src/linux_url_plugin.rs

```diff
@@ -0,0 +1,63 @@
+use log::{info, warn};
+use std::io::Read;
+use std::os::unix::fs::PermissionsExt;
+use std::os::unix::net::UnixListener;
+use std::time::Duration;
+
+use tauri::{
+    async_runtime::spawn,
+    plugin::{Builder, TauriPlugin},
+    Manager, Runtime,
+};
+use tokio::time::sleep;
+
+pub(crate) const OCKAM_OPEN_URL_SOCK: &str = "/tmp/.ockam-open-url-sock";
```

Comment:
I wonder if this should use [$XDG_RUNTIME_DIR or $XDG_CONFIG_HOME](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html#basics) rather than assuming these:
- The system `TMPDIR` is `/tmp` (that's a symlink on MacOS, so it usually still works in practice)
- The running user definitely has write access to the root of `$TMPDIR`
- No one malicious also has simultaneous write access to `OCKAM_OPEN_URL_SOCK` path while it exists

You've already defended against the third concern though 👍🏻

Reply:
Agree, seems like `XDG_RUNTIME_DIR` is a better choice overall :+1: